Item/Item.module.less

```diff
@@ -156,38 +172,55 @@
 			}
 		}
 
+		.disabled({
+			opacity: @agate-item-disabled-opacity;
+			color: @agate-item-disabled-color;
+			border-color: @agate-item-disabled-border-color;
+		});
+
 		.focus({
 			border-color: @agate-item-focus-border-color;
 			background-color: @agate-item-focus-bg-color;
+			background-image: @agate-item-focus-bg-image;
 			color: @agate-item-focus-color;
 
 			.itemContent {
 				.label {
 					color: @agate-item-label-color;
 				}
 			}
+
+			&.selected {
+				&::before {
+					background-color: @agate-indicator-focus-bg-color;
+				}
+			}
 		});
 
 		&.selected {
 			background-color: @agate-item-selected-bg-color;
 			border-color: @agate-item-selected-border-color;
 			color: @agate-item-focus-color;
+			padding: @agate-item-selected-padding;
 
 			&::before {
 				content: "";
 				height: @agate-indicator-height;
-				width: @agate-indicator-height;
+				min-width: @agate-indicator-height; // `min-width` is used to ensure that the indicator has a correct width with default item
 				background-color: @agate-indicator-bg-color;
 				border-radius: @agate-indicator-border-radius;
 				align-self: @agate-indicator-align-self;
 				margin-right: @agate-indicator-margin-right;
 			}
 
-			.itemContent {
-				.label {
-					color: @agate-item-selected-label-color;
+			.enact-locale-rtl({
+				padding: @agate-item-selected-rtl-padding;
+
+				&::before {
+					margin-left: @agate-indicator-rtl-margin-left;
+					margin-right: @agate-indicator-rtl-margin-right;
```

Comment:
this can be simplified. use padding-start-end() at line 204 and margin-start-end at line 213()